lib/connection.js

```diff
@@ -333,7 +333,8 @@ Connection.prototype.request = function(request, options, callback) {
 Connection.prototype.requestGet = function(url, options, callback) {
   var request = {
     method: "GET",
-    url: url
+    url: url,
+    headers: options.headers
```

Comment:
As the `options` argument is for `HttpApi` configuration, the http headers info is not to be here.